src/status_im/chat/views/input/send_button.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im.chat.views.input.send-button
+  (:require-macros [status-im.utils.views :refer [defview letsubs]])
+  (:require [clojure.string :as string]
+            [reagent.core :as reagent]
+            [re-frame.core :as re-frame]
+            [status-im.chat.styles.input.send-button :as style]
+            [status-im.ui.components.animation :as animation]
+            [status-im.ui.components.react :as react]
+            [status-im.ui.components.icons.vector-icons :as vi]
+            [status-im.utils.utils :as utils]
+            [taoensso.timbre :as log]))
+
+(defn send-button-view-on-update [{:keys [spin-value opacity-value command-completion]}]
+  (fn [_]
+    (let [to-spin-value (if (some #{:complete :no-command} [@command-completion]) 1 0)]
+      (animation/start
+        (animation/timing spin-value {:toValue  to-spin-value
+                                      :duration 300})))))
+
+(defview send-button-view []
+  (letsubs [command-completion [:command-completion]
+            selected-command   [:selected-chat-command]
+            input-text         [:chat :input-text]
+            seq-arg-input-text [:chat :seq-argument-input-text]
+            spin-value         (animation/create-value 1)
+            on-update          (send-button-view-on-update {:spin-value         spin-value
+                                                            :command-completion command-completion})]
+    {:component-did-update on-update}
+    (let [{:keys [hide-send-button sequential-params]} (:command selected-command)]
+      (when (and (not (string/blank? input-text))
+                 (or (not selected-command)
+                     (some #{:complete :less-than-needed} [command-completion]))
+                 (not hide-send-button))
+        [react/touchable-highlight {:on-press #(if sequential-params
+                                                 (do
+                                                   (when-not (string/blank? seq-arg-input-text)
+                                                     (re-frame/dispatch [:send-seq-argument]))
+                                                   (utils/set-timeout
+                                                     (fn [] (re-frame/dispatch [:chat-input-focus :seq-input-ref]))
+                                                     100))
```

Comment:
Can you add a comment detailing the need for timeout?

Reply:
No. It was here before, and I haven't touched it. Simply moved from `status-im.chat.views.input.input` ns to a new one.